### ISSUE TITLE
Update the USWDS anchor component to address USAgov feedback

### ISSUE THREAD

**Issue by afeijoo** [uswds/uswds#3525]
@mariamarrero provided the following feedback in [Digital.gov #2351](https://github.com/GSA/digitalgov.gov/issues/2351). I moved Maria's feedback to this USWDS repo so we can track it in our project. 

+++++

### Why we really like the "anchor" idea

- If implemented as part of the USWDS, it will add some nice consistency to how agencies link back to USA.gov. A huge plus on this one!
- Making the requirement that states that all gov sites need to link to USA.gov "official" in the USWSDS (with consistent language in both, English and Spanish) is fantastic. Other gov agencies won't have to worry about where exactly to put it, the verbiage and the style.

### Linking to USA.gov home page as opposed to the "contact us" page

- Interactions with the USA.gov contact center cost $, and our general strategy is to push self-service first, especially for the chat service (which is more expensive than phone on the current contract.) In fact, a couple of weeks ago, we made some changes to our IVR (Interactive Voice Response) system that people get when they call our contact center with 2 main goals: improve efficiency and contain costs.
- People that go to the home page first can search and perhaps find an answer on their own before jumping in and chatting or calling us right away. 
- Most people come to USA.gov with a specific task in mind, but we have observed in usability testings that as people navigate through our website, they also come across important information they did not expect to find and mention how valuable that is.
- Setting expectations on what USA.gov contact center agents can help with is key as part of providing a better user experience for our visitors. We don't want to overpromise and get inundated with contacts that we cannot afford and with questions that are best served by the original agency. For example: If someone needs to know about their tax refund, calling the IRS directly is their best choice. For other complicated questions where people don't know what agency they need to reach, our agents can help.

### Changes to the text and call to action link

- Our recommendation is to modify the text to "Looking for U.S. government and services? Visit USA.gov".
- I am not sure if the USWDS plans to provide guidance for government websites in Spanish like the SSA website, but if so, the text would be "¿Busca información y servicios del Gobierno de Estados Unidos? Visite USAGov en Español".

### Icon vs no icon and button color

1. We are recommending to remove any icon/graphic next to the "anchor" text for the following reasons:

- Each gov site has its own style (color and look and feel). The icon may not match their branding against a given icon and may want to make modifications to make it fit their branding. That includes different colors, shapes, sizes, etc. Therefore, it opens the doors for inconsistency, which would be the opposite of what the USWDS is trying to accomplish.

- In this case, the content and the link (call the action) are clear and the icon would be there mainly for decoration (based on the new recommended text)

2. The blue color (#2491FF) of the current button does not pass color contrast (on WebAIM contrast checker) for normal text. An alternative color (back text on white background) will work better based on the overall color palette in Digital.gov. 

Finally, we put together a couple of concepts that might work for Digital.gov that reflects our feedback. They are attached. 

Our team's preferred concept is the one with no button  (link_concept.png and mobile_link_concept.png) for the following reasons:

- It reflects a stronger visual connection with the text next to it due to the same visual treatment.
It is not overpowering and competing with other important information (for example, other calls to action) on footers. 
- Again, when it comes to styles and colors on gov websites, this may avoid room for possible inconsistencies as websites might try to apply their own style to it. 
- From previous experience, numerous agencies have requested our logos in different colors and format versions to match their footer requirements. Avoiding the placement of icons or images will help us reduce future requests for assistance with the customization of images from agencies. Especially from those who lack a lot of design resources.

Let me know if you would like to further discuss any of our feedback. We are happy to meet :)

Thanks!

<img width="414" alt="mobile_link_concept" src="https://user-images.githubusercontent.com/6410374/79012646-c4e3ca00-7b34-11ea-845b-414d060d0279.png">
<img width="414" alt="mobile_button_concept" src="https://user-images.githubusercontent.com/6410374/79012656-cca36e80-7b34-11ea-87f1-6b94f7b8a683.png">

![link_concept](https://user-images.githubusercontent.com/6410374/79012731-f78dc280-7b34-11ea-8e6d-8c1475a2ccbb.png)
![button_concept](https://user-images.githubusercontent.com/6410374/79012759-12603700-7b35-11ea-9715-c42fffd6af5c.png)

**Comment by afeijoo** [comment]
@mariamarrero — can you provide us with a custom URL endpoint that we should link to, and you can then use a `302 redirect` to point it where you need over time. At this moment, it might be homepage, but in the future, you might want it to be a different portal 

_(just make sure it is a `302 redirect` and not a `301 redirect`)_

_Originally posted by @jeremyzilar in https://github.com/GSA/digitalgov.gov/issues/2351#issuecomment-612161091_

---

Also, we should probably also include the domain or sub-domain as a query param to the URL s (e.g. `https://usa.gov/example/?digital.gov` or `https://usa.gov/example/?designsystem.digital.gov`) so that you can be in a better position to do something conditional based off the site they are coming from.

Thoughts on this?

_Originally posted by @jeremyzilar in https://github.com/GSA/digitalgov.gov/issues/2351#issuecomment-612163341_

---

@jeremyzilar, I am going to defer those questions to Russell O'Neil and David Kaufmann. They are better suited to answer them. Stan tunned.

_Originally posted by @mariamarrero in https://github.com/GSA/digitalgov.gov/issues/2351#issuecomment-612961560_

---

@jeremyzilar I really like your thinking on this. On occasion, we have briefly pondered doing something similar. We know that people who start their usa.gov session on our home page and are referred by a different dot gov give us lower csat scores than most other segments. This segment accounts for about 1.5%ish of our traffic, which isn't nothing. However, when you start to break it out by individual referring websites, the slices get smaller right away. That said, one could make a case for wanting to try something based on the above.

But IMHO I think for now at least we should go ahead with just the regular home page link for the footer (i.e. no redirect), but we can append it with a single casual parameter (i.e. ?source=uswds-footer).

My thinking is:
1) We could add the redirect complexity later if necessary in later USWDS versions
2) Some types of redirects can mess up the referral info we already get in GA, which is high quality
3) We don't know what we would actually do to be helpful for people from these other websites. That would take some investigation, which we can't do right now
4) If we decided to pilot a redirect for, say, people going from irs.gov to our home page, we could do a javascript redirect using Qualtrics, which lets us use referral url as part of the condition. So we could try something out that way if we could decide upon information that we feel would provide a better experience than our home page. Actually, we could fire a modal with some information in it instead of a redirect, if we desired.
5) The casual query parameter suggested above could be used in an unfiltered view in GA to segment footer referrals, or for isolating a javascript redirect to footer referrals only.

Thanks for thinking this all through. Really helpful!

_Originally posted by @DK1999 in https://github.com/GSA/digitalgov.gov/issues/2351#issuecomment-613503220_
